internal/gps/source_manager.go

```diff
@@ -87,8 +87,19 @@ type ProjectAnalyzer interface {
 	// expected files containing Manifest and Lock data are merely absent.
 	DeriveManifestAndLock(path string, importRoot ProjectRoot) (Manifest, Lock, error)
 
-	// Report the name and version of this ProjectAnalyzer.
-	Info() (name string, version int)
+	// Info reports this project analyzer's info.
+	Info() ProjectAnalyzerInfo
+}
+
+// ProjectAnalyzerInfo holds name and version information for a ProjectAnalyzer.
```

Comment:
small modification: "ProjectAnalyzerInfo indicates a ProjectAnalyzer's name and version."